### PR TITLE
Ov 13 ut schema tools cull nested columns

### LINKED ISSUE #393: [TESTS] - SchemaTools - cullNestedColumns

### PR DESCRIPTION
**Pull request by sundarshankar89**
Ready for review. OV-13 UT for Schema Tools Cull Nested Columns

closes #393

### CODE REVIEW

**Comment by sundarshankar89** [comment]
@sawankulkarni-db or @Sriram-databricks can you take a look at this.

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala [hunk 1]
```diff
@@ -106,6 +109,15 @@ object SchemaTools extends SparkSessionWrapper {
    * @return
    */
   def cullNestedColumns(df: DataFrame, structToModify: String, nestedFieldsToCull: Array[String]): DataFrame = {
+    //Exception Block
+    if (nestedFieldsToCull.filter(_.contains(".")).size != 0) {
```
Comment (GeekSheikh) [inline src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:113]: switch to 
`nestedFieldsToCull.exists(_.contains("."))`

File: src/test/scala/com/databricks/labs/overwatch/utils/StringExtTest.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.databricks.labs.overwatch.utils
+
+import com.databricks.labs.overwatch.SparkSessionTestWrapper
+import com.databricks.labs.overwatch.utils.StringExt.StringHelpers
+import org.scalatest.funspec.AnyFunSpec
+
+class StringExtTest extends AnyFunSpec with SparkSessionTestWrapper {
+
+  describe("StringHelpers Test") {
```
[the remark this answers is not in the document]
Comment (sundarshankar89) [reply src/test/scala/com/databricks/labs/overwatch/utils/StringExtTest.scala:8]: Updated in the new commit

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala (hunk 1 above)
Comment (sundarshankar89) [reply src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala]: Modified

**Comment by sundarshankar89** [comment]
@GeekSheikh Ready for review.

**Review by GeekSheikh — APPROVED** [review]
LGTM

**Pull request merged**